1.py

```diff
@@ -0,0 +1,70 @@
+import logging
+import datetime
+import argparse
+from datetime import datetime
+from collections import namedtuple
+logging.basicConfig(filename="log1.log", encoding="utf8", level=logging.INFO) 
+logger = logging.getLogger("log") 
```

Comment:
Пожалуйста, приводите код к виду PEP8 
```suggestion
import logging
import datetime
import argparse
from datetime import datetime
from collections import namedtuple

logging.basicConfig(filename="log1.log", encoding="utf8", level=logging.INFO)
logger = logging.getLogger("log")
```